src/tokenizer/types.js

```diff
@@ -152,7 +152,7 @@ export const keywords = {
   "new": new KeywordTokenType("new", { beforeExpr, startsExpr }),
   "this": new KeywordTokenType("this", { startsExpr }),
   "super": new KeywordTokenType("super", { startsExpr }),
-  "class": new KeywordTokenType("class"),
+  "class": new KeywordTokenType("class", { startsExpr }),
```

Comment:
This should be in a separate PR (seems like something that should go upstream? would first want to understand why it wasn't here before...)